leaderboard.py

```diff
@@ -455,13 +483,29 @@ def generate_memer_board(self, ctx, method, lim, span):
             memers = self.session.query(Message.user_id, (cast(func.sum(Message.rx1_count), Float) + self.stat_dist_data['alpha']) / (cast(func.sum(Message.rx1_count), Float) + cast(func.sum(Message.rx2_count), Float) + self.stat_dist_data['alpha'] + self.stat_dist_data['beta'])).filter(
                 Message.server_id == db_server.id, Message.created_at > start_date).group_by(Message.user_id).order_by(((cast(func.sum(Message.rx1_count), Float) + self.stat_dist_data['alpha']) / (cast(func.sum(Message.rx1_count), Float) + cast(func.sum(Message.rx2_count), Float) + self.stat_dist_data['alpha'] + self.stat_dist_data['beta'])).desc()).limit(message_count).all()
             heading = 'Top ' + heading + ' by beta distribution'
-        else: # if (method == 'beta_down'):
+        elif (method == 'beta_down'):
             if (self.stat_dist_data is None or (datetime.now() - self.stat_dist_data['timestamp']).days > 14):
                 self.refresh_beta_dist(db_server)
             memers = self.session.query(Message.user_id, (cast(func.sum(Message.rx1_count), Float) + self.stat_dist_data['alpha']) / (cast(func.sum(Message.rx1_count), Float) + cast(func.sum(Message.rx2_count), Float) + self.stat_dist_data['alpha'] + self.stat_dist_data['beta'])).filter(
                 Message.server_id == db_server.id, Message.created_at > start_date).group_by(Message.user_id).order_by(((cast(func.sum(Message.rx1_count), Float) + self.stat_dist_data['alpha']) / (cast(func.sum(Message.rx1_count), Float) + cast(func.sum(Message.rx2_count), Float) + self.stat_dist_data['alpha'] + self.stat_dist_data['beta'])).asc()).limit(message_count).all()
             heading = 'Shit ' + heading + ' by beta distribution'
-        
+        elif (method == 'compare_up'):
+            memers = self.session.query(func.sum(Message.rx1_count), func.sum(Message.rx2_count)).filter(
+                Message.server_id == db_server.id, Message.created_at > start_date, Message.user_id == db_user.id).group_by(Message.user_id).all()
```

Comment:
you don't have the `db_user` object, so you'll have to get it first before using it.